backend/app/assets/stylesheets/spree/backend/shared/_layout.scss

```diff
@@ -14,6 +14,9 @@ body {
 .admin-nav {
   @include position(absolute, 0 null 0 0);
   width: $width-sidebar;
+  img {
+    max-width: 100%;
+  }
 }
```

Comment:
I don't think this is a concern. We use descendant selectors liberally in our styles. A `> img` here also won't achieve the desired effect here.

From the linked document

> Note: This document was originally written in 2000. Much has changed when it comes to writing CSS that is fast. This guide is not an accurate representation of the bottlenecks in the browser's rendering pipeline.